### ISSUE TITLE
Uncaught TypeError: Class constructor FuzzyFinderView cannot be invoked without 'new'

### ISSUE THREAD

**Issue by viddo** [viddo/recent-files-fuzzy-finder#30]
https://github.com/atom/atom/releases/tag/v1.26.0-beta0 contains an update which contains to breaking changes for this package:

1. https://github.com/atom/fuzzy-finder/pull/337 - Removing babel, which makes the babel-ES6-class `extend` to error out (see title of this issue). Should be fixable by removing any babel transformations here too.
2. https://github.com/atom/fuzzy-finder/pull/335 - Telemetry feature includes modifications to internal method APIs, no longer expecting paths but richer object items. Should be fixable by looking through the changes in this diff.

Most likely will have to introduce the `semver` check on the same lines as in #21 earlier on to make things compatible for further upcoming stable and beta releases.

---

**Atom**: 1.26.0-beta0 x64
**Electron**: 1.7.11
**OS**: Mac OS X 10.12.6
**Thrown From**: [recent-files-fuzzy-finder](https://github.com/viddo/recent-files-fuzzy-finder) package 0.4.0

### Stack Trace

Uncaught TypeError: Class constructor FuzzyFinderView cannot be invoked without 'new'

```
At /Users/viddo/.atom-beta/packages/recent-files-fuzzy-finder/lib/recent-files-view.js:8

TypeError: Class constructor FuzzyFinderView cannot be invoked without 'new'
    at /packages/recent-files-fuzzy-finder/lib/recent-files-view.js:8:37)
    at Object.createRecentFilesView (/packages/recent-files-fuzzy-finder/lib/main.coffee:36:30)
    at /packages/recent-files-fuzzy-finder/lib/main.coffee:17:54)
    at CommandRegistry.handleCommandEvent (/Applications/Atom Beta.app/Contents/Resources/app/src/command-registry.js:384:49)
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom Beta.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:621:22)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom Beta.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:412:28)
    at WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom Beta.app/Contents/Resources/app/src/window-event-handler.js:110:40)
```

### Commands

```
     -0:25.9.0 textual-velocity:start-session (atom-workspace.workspace.scrollbars-visible-always)
     -0:05.0 recent-files-fuzzy-finder:toggle-finder (atom-pane.pane)
```

**Comment by nguyenhose** [comment]
hi, I still got this error and seem the fixed patch still isn't published

**Comment by viddo** [comment]
@nguyenhose The patch for this error was released in v0.5
Next time you see that error popup please report it as a new issue or paste the context info (there should be a button/link to copy the stack and context info), so we can investigate further.
